src/controllers/idp.ts

```diff
@@ -15,6 +16,7 @@ import {
 import { SelfCareIdentity } from "../auth-strategies/selfcare_identity_strategy";
 import { createSessionToken } from "../auth-strategies/selfcare_session_strategy";
 import { selfcareSessionCreds } from "../config";
+import * as config from "../config";
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏try not to import twice from the same package

Reply:
f822088